[CI:DOCS] podman-generate-systemd.1.md: document --sdnotify (#15043)
containers/podman: docs/source/markdown/podman-generate-systemd.1.md

_Note: If you use this command with the remote client, including Mac and Windows (excluding WSL2) machines, you would still have to place the generated units on the remote system.  Moreover, please make sure that the XDG_RUNTIME_DIR environment variable is set.  If unset, you may set it via `export XDG_RUNTIME_DIR=/run/user/$(id -u)`._

> _Note: The generated `podman run` command contains an `--sdnotify` option with the value taken from the container, unless the container value was set to __ignore__, then the value __conmon__ is used. If the container does not have any explicitly set value, the value __conmon__ is used._

Reviewer: ```suggestion
_Note: The generated `podman run` command contains an `--sdnotify` option with the value taken from the container. If the container does not have any explicitly set value or the value is set to __ignore__, the value __conmon__ is used._
```
